src/javaExercises/Palindrome/Palindrome.java

```diff
@@ -0,0 +1,49 @@
+package javaExercises.Palindrome;
+
+import java.util.Scanner;
+
+public class Palindrome {
+    public static boolean checkIfPalindrome(String text) {
+        String cleanString = text.replaceAll("\\s+", "");
+        int length = cleanString.length();
+
+        int forward = 0;
+        int backward = length - 1;
+
+        while (backward > forward) {
+
+            char charForward = cleanString.charAt(forward++);
+            char charBackward = cleanString.charAt(backward--);
+
+            if (charBackward != charForward)
+                return false;
+        }
+        return true;
+    }
+
+    public static boolean verifyIfPalindrome(String text) {
+
+        StringBuilder reversedString = new StringBuilder();
+        String cleanString = text.replaceAll("\\s+", "");
```

Comment:
there's a `trim()` method for this, don't get complicated

Reply:
Done